nemo/collections/asr/parts/conformer_modules.py

```diff
@@ -38,13 +38,14 @@ def __init__(
         self,
         d_model,
         d_ff,
-        conv_kernel_size,
-        self_attention_model,
-        n_heads,
-        dropout,
-        dropout_att,
-        pos_bias_u,
-        pos_bias_v,
+        self_attention_model='rel_pos',
+        n_heads=4,
+        conv_kernel_size=31,
+        conv_norm_type="batch_norm",
```

Comment:
Same as above - maybe `batch` for consistency

Reply:
dropped the feature.